packages/kit-headless/src/components/select/select.tsx

```diff
@@ -113,9 +118,9 @@ export const SelectRoot = component$(
   }
 );
 
-export interface TriggerProps extends StyleProps {
+export type TriggerProps = {
   disabled?: boolean;
```

Comment:
Do we still need this?
I guess `disabled`is part of `QwikIntrinsicElements['button']`.

Reply:
I believe so. I added this with marcus on a short call.